volume/drivers/api.go

```diff
@@ -4,11 +4,13 @@ package volumedrivers
 
 import "github.com/docker/docker/volume"
 
+// NewVolumeDriver returns a driver has the given name mapped on the given client.
 func NewVolumeDriver(name string, c client) volume.Driver {
 	proxy := &volumeDriverProxy{c}
 	return &volumeDriverAdapter{name, proxy}
 }
 
+// VolumeDriver manages volumes from end to end.
```

Comment:
VolumeDriver in this context is explicitly for facilitating communications with plugins, and that is all.